.github/workflows/test-e2e.yml

```diff
@@ -28,7 +28,7 @@ env:
   DATADOG_TRACE_NEXTJS_TEST: true
   DATADOG_API_KEY: foo
   TEST_CONCURRENCY: 2
-  NEXT_E2E_TEST_TIMEOUT: 300000
+  NEXT_E2E_TEST_TIMEOUT: 600000
```

Comment:
Similarly as above - not related to deleting deploys, but some of the test suites seemed to be on the verge of timeouts and would sometimes go over so this _might_ reduce some of the flakiness (not very confident in that part due to troubles with reproducing problems locally or even when running individual tests in github actions and not locally, but I think it's worth a try)